### ISSUE TITLE
Help with configs

### ISSUE THREAD

**Issue by Disketaa** [B1n-ry/Youre-in-grave-danger#50]
I'm using the last version of this mod for 1.18.2 and sometimes items are doesn't drops at all from the grave. Also players can't destroy other graves in multiplayer, but "EnableRobbing" is on. Maybe something wrong with the config? Thanks

```
[graveSettings]
generateGraves = true
retrievalType = "ON_BREAK"
dropType = "ON_GROUND"
dropPlayerHead = false
dropGraveBlock = false
requireGraveItem = false
generateEmptyGraves = false
ignoreSpawnProtection = true
deathInSpawnProtection = "CREATE_GRAVE"
priority = "GRAVE"
unlockableGraves = true
deleteEnchantments = ["minecraft:vanishing_curse"]
soulboundEnchantments = ["yigd:soulbound"]
applyBindingCurse = true
trySoft = true
tryStrict = true
replaceWhenClaimed = true
blacklistDimensions = []
graveInVoid = false
ignoreDeathTypes = []
graveSpawnHeight = 2
lastResort = "DROP_ITEMS"
putXpInGrave = false
defaultXpDrop = false
xpDropPercent = 60
tellDeathPos = false
maxGraveBackups = 50

[graveSettings.graveRobbing]
enableRobbing = true
onlyMurderer = false
afterTime = 0
timeType = "HOURS"
robPriority = "INVENTORY"

[graveSettings.graveDeletion]
canDelete = false
afterTime = 1
timeType = "HOURS"
dropInventory = true

[graveSettings.itemLoss]
enableLoss = false
ignoreSoulboundItems = true
affectStacks = false
usePercentRange = false
lossRangeFrom = 0
lossRangeTo = 5
percentChanceOfLoss = 100

[graveSettings.blockUnderGrave]
generateBlockUnder = true
inOverWorld = "minecraft:cobblestone"
inNether = "minecraft:soul_soil"
inTheEnd = "minecraft:end_stone"
inCustom = "minecraft:dirt"

[graveSettings.graveCompatConfig]
levelzXpInGraves = true
levelzXpDropPercent = 100
claimRuleOverride = "CREATE_GRAVE"

[graveSettings.graveRenderSettings]
useRenderFeatures = true
renderGraveSkull = true
renderGraveOwner = true
textShadow = false
adaptRenderer = true
glowingGrave = true
glowMinDistance = 0
glowMaxDistance = 8
[utilitySettings]
soulboundEnchant = false
deathSightEnchant = false

[utilitySettings.scrollItem]
scrollType = "DISABLED"
retrieveOnRespawn = true

[utilitySettings.graveKeySettings]
enableKeys = false
alwaysRequire = false
retrieveOnRespawn = true
getFromGui = false
rebindable = false
graveKeySpecification = "PLAYER"
[commandToggles]
coreCommandName = "yigd"
retrieveGrave = true
robGrave = true
selfView = true
adminView = true
moderateGraves = true
clearGraveBackups = true
whitelist = true
whitelistAdd = true
whitelistRemove = true
whitelistToggle = true

```

**Comment by B1n-ry** [comment]
It looks all good to me. Unfortunately I can't test this until about a week from now. You could try to get logs of this happening, and see if there's anything out of the ordinary. In the meantime I'm afraid I can't be of much further help though

**Comment by B1n-ry** [comment]
Should be fixed in latest 1.18.2 and 1.19 versions. Sorry it took so long to get back to you with a new release
